<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StuOnDutyCellWidget</class>
 <widget class="QWidget" name="StuOnDutyCellWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>135</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnAdd">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../assets/ClassAdmin.qrc">
         <normaloff>:/img/add.png</normaloff>:/img/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRemove">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../assets/ClassAdmin.qrc">
         <normaloff>:/img/subtract.png</normaloff>:/img/subtract.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../assets/ClassAdmin.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>StuOnDutyCellWidget</receiver>
   <slot>onAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>805</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemove</sender>
   <signal>clicked()</signal>
   <receiver>StuOnDutyCellWidget</receiver>
   <slot>onRemove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>629</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAdd()</slot>
  <slot>onRemove()</slot>
 </slots>
</ui>
